app/views/setting.html

```diff
@@ -14,31 +9,31 @@
           <label for="condensedItems" title="Items that do not match filter criteria will be hidden.">Hide Unfiltered Items while Filtering</label>
         </td>
         <td>
-          <input type="checkbox" ng-model="vm.settings.hideFilteredItems" ng-change="vm.save('hideFilteredItems')" />
+          <input type="checkbox" ng-model="vm.settings.hideFilteredItems" ng-change="vm.save()" />
```

Comment:
not a big thing, but in this case would it make sense to just create a watcher in the controller for `vm.settings` and if it changes, call `vm.save()`? or is it more obvious that a save is happening to show it here on the `ng-change` event?

either way, love this PR. `dimSettingsService.factory.js` looks great now, utilizing filters and such seems to be much cleaner and read better.